app-crypt/moolticute/moolticute-0.42.1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/mooltipass/moolticute.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/mooltipass/moolticute/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm"
+fi
+
+inherit xdg-utils qmake-utils udev
+
+DESCRIPTION="Mooltipass crossplatform daemon/tools"
+HOMEPAGE="https://github.com/mooltipass/moolticute"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	>=dev-libs/libusb-1.0.20
+	dev-qt/qtdbus:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qttest:5
+	dev-qt/qtwebsockets:5
+	dev-qt/qtwidgets:5
+"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
```

Comment:
`linguist-tools` should be in BDEPEND. Sorry I didnt see this the first time. Could you also update the -9999 ebuild to reflect this :)

Reply:
Also I'd prefer if you split editing -9999 into its own commit.

Reply:
Indeed, I forgot that change. Thanks!!